color_code_test.py

```diff
@@ -0,0 +1,25 @@
+from main import * 
+
+MAJOR_COLORS_LIST = ['White', 'Red', 'Black', 'Yellow', 'Violet']
+MINOR_COLORS_LIST = ["Blue", "Orange", "Green", "Brown", "Slate"]
+
+def test_number_to_pair(pair_number,
+                        expected_major_color, expected_minor_color):
+  major_color, minor_color = get_color_from_pair_number(pair_number, MAJOR_COLORS_LIST, MINOR_COLORS_LIST)
+  assert(major_color == expected_major_color)
+  assert(minor_color == expected_minor_color)
+
+
+def test_pair_to_number(major_color, minor_color, expected_pair_number):
+  pair_number = get_pair_number_from_color(major_color, minor_color, MAJOR_COLORS_LIST, MINOR_COLORS_LIST)
+  assert(pair_number == expected_pair_number)
+
+
+if __name__ == '__main__':
+  test_number_to_pair(4, 'White', 'Brown')
+  test_number_to_pair(5, 'White', 'Slate')
+  test_pair_to_number('Black', 'Orange', 12)
+  test_pair_to_number('Violet', 'Slate', 25)
+  test_pair_to_number('Red', 'Orange', 7)
+  print_reference_manual(MAJOR_COLORS_LIST, MINOR_COLORS_LIST)

```

Comment:
The above tests have asserts, so it's not needed to check their output manually. However, `print_reference_manual` needs 25 lines of output to be inspected manually. Can you think of asserts to reduce the manual test effort?